pkg/gateway/multiparts/tracker.go

```diff
@@ -22,14 +26,36 @@ type MultipartUpload struct {
 	ContentType     string    `db:"content_type"`
 }
 
+func (m Metadata) Set(k, v string) {
+	m[strings.ToLower(k)] = v
+}
+
+func (m Metadata) Get(k string) string {
+	return m[strings.ToLower(k)]
+}
+func (m Metadata) Value() (driver.Value, error) {
+	return json.Marshal(m)
+}
+
+func (m *Metadata) Scan(src interface{}) error {
```

Comment:
Why not accept the concrete type `[]byte`?

Reply:
This code was not touched - just moved in the file.
If we change this interface we will need also to touch the tracker usage which I was very careful not to disturb